wagtail/contrib/forms/tests/test_views.py

```diff
@@ -361,6 +362,52 @@ def setUp(self):
         # Login
         self.login()
 
+    def get_url(self, viewname, *args, **kwargs):
+        """
+        Reverse a URL and append querystring arguments.
+        """
+        querystring = kwargs.pop("querystring", None)
+        url = reverse(viewname, args=args, kwargs=kwargs)
+        if querystring:
+            url += "?" + urlencode(querystring)
+        return url
```

Comment:
Looking at how this is used in the test, there doesn't seem to be much value in extracting this function, as we'll have to specify the `viewname`, `page_id`, and `querystring`, which are all very specific to this method's implementation. Let's remove this to avoid adding implementation-like code inside tests.